stf/test/saros/stf/test/stf/menubar/SarosPreferencesTest.java

```diff
@@ -37,6 +24,10 @@ public void beforeEveryTest() throws Exception {
     ALICE.controlBot().getAccountManipulator().deleteAllAccounts();
   }
 
+  /*
```

Comment:
Either delete them or set them to ignore. The class is named SarosPreferencesTest. These test cases do not test anything from the preference page.